shell/bwrap.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -ue
+
+# Configure with the following lines in ~/.opam/config:
+#
+#   wrap-build-commands:   ["%{root}%/opam-init/hooks/bwrap.sh" "build"]
+#   wrap-install-commands: ["%{root}%/opam-init/hooks/bwrap.sh" "install"]
+#   wrap-remove-commands:  ["%{root}%/opam-init/hooks/bwrap.sh" "remove"]
+
+if ! command -v bwrap >/dev/null; then
+    echo "The 'bwrap' command was not found. Install it, or disable sandboxing in" >&2
```

Comment:
Since bubblewrap is a new tool, perhaps add a URL to its source here?  The link between `bwrap` and Bubblewrap wouldn't be clear otherwise

Reply:
Good point!